packages/nimbus/electron/server/utils/mock/generateUserProfile.ts

```
@@ -0,0 +1,83 @@
+        "$createdBy": () => chance.name(),
+        "$updatedBy": () => chance.name(),
+        "$middleName": () => chance.first(),
+        "$emergencyContactPhone": () => chance.phone(),
+        "$address2": () => chance.street(),
+    };
+    
+    return fieldMap[schemaFieldQueryString]?.() || null; // Return null for unknown fields
+    
+
+    // Return the generated data based on the schema field query string
+    return fieldMap[schemaFieldQueryString]?.() || null; // Return null for unknown fields
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)